sa/_db-next/migrations/20170718114448_DBCleanup.sql

```diff
@@ -0,0 +1,61 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+DROP TABLE externalCerts;
+
+DROP TABLE identifierData;
+
+ALTER TABLE certificateStatus DROP LockCol;
```

Comment:
Is the same true of subscriberApproved? Also, the PR description should document that deploy order dependency (even though it is indeed the default).